packages/terra-form-input/tests/jest/FormInput.test.jsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import FormInput from '../../src/FormInput';
+
+describe('FormInput', () => {
+  const defaultRender = <FormInput />;
+
+  // Snapshot Tests
+  it('should render a default component', () => {
+    const wrapper = shallow(defaultRender);
+    expect(wrapper).toMatchSnapshot();
+  });
+
+  // Prop Tests
+  it('should use the default value when no value is given', () => {
+    const wrapper = shallow(defaultRender);
+    expect(wrapper.find('.form-input').text()).toEqual('');
```

Comment:
Do we need this test? I feel that your snapshot tests are sufficient enough

Reply:
I think this was one of the auto-generated tests, removed it. 297550fd